crates/bitwarden-core/src/key_management/mod.rs

```diff
@@ -24,9 +24,11 @@ pub use crypto_client::CryptoClient;
 #[cfg(feature = "internal")]
 mod master_password;
 #[cfg(feature = "internal")]
+pub use master_password::MasterPasswordAuthenticationData;
+#[cfg(feature = "internal")]
 pub use master_password::MasterPasswordError;
 #[cfg(feature = "internal")]
-pub(crate) use master_password::{MasterPasswordAuthenticationData, MasterPasswordUnlockData};
+pub use master_password::MasterPasswordUnlockData;
```

Comment:
Nit: These have the same feature compile time flag, and the same package prefix, you can replace with `pub use master_password:{MasterPasswordAuthenticationData, MasterPasswordError, MasterPasswordUnlockData}`